docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Categorical/OneHotEncoding.cs

```diff
@@ -17,53 +16,51 @@ public static void Example()
             // Get a small dataset as an IEnumerable.
             var samples = new List<DataPoint>()
             {
-                new DataPoint(){ Label = 0, Education = "0-5yrs" },
-                new DataPoint(){ Label = 1, Education = "0-5yrs" },
-                new DataPoint(){ Label = 45, Education = "6-11yrs" },
-                new DataPoint(){ Label = 50, Education = "6-11yrs" },
-                new DataPoint(){ Label = 50, Education = "11-15yrs" },
+                new DataPoint(){ Label = 0, Education = "0-5yrs", ZipCode = "98005" },
+                new DataPoint(){ Label = 1, Education = "0-5yrs", ZipCode = "98052" },
+                new DataPoint(){ Label = 45, Education = "6-11yrs", ZipCode = "98005" },
+                new DataPoint(){ Label = 50, Education = "6-11yrs", ZipCode = "98052" },
+                new DataPoint(){ Label = 50, Education = "11-15yrs", ZipCode = "98005" },
             };
 
             // Convert training data to IDataView.
-            var trainData = mlContext.Data.LoadFromEnumerable(samples);
+            var data = mlContext.Data.LoadFromEnumerable(samples);
 
             // A pipeline for one hot encoding the Education column.
-            var bagPipeline = mlContext.Transforms.Categorical.OneHotEncoding("EducationOneHotEncoded", "Education", OutputKind.Bag);
```

Comment:
>", OutputKind.Bag); [](start = 114, length = 19)

I would leave it, so that it makes sense why we call it bagPipeline.

Reply:
am using the default  (which is uses Indicator , not bagging) ..  also renamed it to `pipeline`

---
In reply to: [271811206](https://github.com/dotnet/machinelearning/pull/3179#discussion_r271811206) [](ancestors = 271811206)